packages/zoe/src/zoeService/startInstance.js

```diff
@@ -61,6 +63,18 @@ export const makeStartInstance = (
 
     const instance = makeInstanceHandle();
 
+    const snoozedStrings = [];
+    const isSnoozed = string =>
+      snoozedStrings.some(s => string.indexOf(s) > -1);
```

Comment:
not clear to me why any substring match is sufficient. please explain in jsdoc comment.

Reply:
substring match is gone. Resolving.